lib/licensee/project_files/license_file.rb

```diff
@@ -14,10 +14,13 @@ def attribution
 
       def self.name_score(filename)
         return 1.0 if filename =~ /\A(un)?licen[sc]e\z/i
-        return 0.9 if filename =~ /\A(un)?licen[sc]e\.(md|markdown|txt)\z/i
-        return 0.8 if filename =~ /\Acopy(ing|right)(\.[^.]+)?\z/i
-        return 0.7 if filename =~ /\A(un)?licen[sc]e\.[^.]+\z/i
-        return 0.5 if filename =~ /licen[sc]e/i
+        return 0.9 if filename =~ /\Acopy(ing|right)\z/i
```

Comment:
I think `LICENSE.md` should still take precedence over `COPYING` when both our present (but agree `COPYING` should take precedence over `LICENSE.image`)